src/index.js

```diff
@@ -94,7 +93,6 @@ function App() {
         <SecureRoute path="/datavis" component={ExampleDataViz} />
         <Route component={NotFoundPage} />
       </Switch>
-      <Footer />
```

Comment:
Instead of deleting the footer entirely, maybe we would make the footer a SecureRoute so the user has to be signed in to see it. 
`<SecureRoute path="/" component={Footer} />`

Reply:
I like this idea a lot, Nick!

Reply:
Thanks for the great idea, Nick.